src/components/Tooltip/types.ts

```diff
@@ -22,7 +17,7 @@ export interface TooltipProps {
     position: Position;
 
     /**
-     * Set text to show in tooltip
+     * Set element/text that will be shown within the tooltip
      */
-    text?: string;
+    content?: JSX.Element | string;
```

Comment:
lets make content required? What do you think about it

Reply:
Yea, ur right